src/components/Dialogs/RegionDialog/RegionDialogComponent.tsx

```diff
@@ -98,14 +132,126 @@ export class RegionDialogComponent extends React.Component {
+                                                max={RegionStore.MAX_DASH_LENGTH}
+                                                value={(region as VectorAnnotationStore).pointerLength}
+                                                stepSize={1}
+                                                onValueChange={value => (region as VectorAnnotationStore).setPointerLength(value)}
+                                            />
+                                        </FormGroup>
+                                        <FormGroup inline={true} label="Arrow Tip Width" labelInfo="(px)">
+                                            <SafeNumericInput
+                                                placeholder="Width"
+                                                min={0}
+                                                max={RegionStore.MAX_DASH_LENGTH}
+                                                value={(region as VectorAnnotationStore).pointerWidth}
+                                                stepSize={1}
+                                                onValueChange={value => (region as VectorAnnotationStore).setPointerWidth(value)}
+                                            />
+                                        </FormGroup>
+                                    </div>
+                                )}
+                            </AppearanceForm>
                             <LineRegionForm region={region} frame={frame} wcsInfo={frame.validWcs ? frame.wcsInfoForTransformation : 0} />
                         </React.Fragment>
                     );
                     editableRegion = true;
                     break;
+                case CARTA.RegionType.ANNCOMPASS:
+                case CARTA.RegionType.ANNRULER:
+                    bodyContent = (
+                        <React.Fragment>
+                            <AppearanceForm region={region} darkTheme={appStore.darkTheme}>
+                                {region.regionType === CARTA.RegionType.ANNCOMPASS && (
+                                    <div className="form-contents">
+                                        <FormGroup inline={true} label="Show North Arrowhead">
+                                            <Switch
+                                                checked={(region as CompassAnnotationStore).northArrowhead}
+                                                onChange={(ev: React.ChangeEvent<HTMLInputElement>) => (region as CompassAnnotationStore).setNorthArrowhead(ev.target.checked)}
+                                            />
+                                        </FormGroup>
+                                        <FormGroup inline={true} label="Show East Arrowhead">
+                                            <Switch checked={(region as CompassAnnotationStore).eastArrowhead} onChange={(ev: React.ChangeEvent<HTMLInputElement>) => (region as CompassAnnotationStore).setEastArrowhead(ev.target.checked)} />
```

Comment:
Only comment I have is that I think we should rather use a dropdown here for arrowheads (north, east, both)

![image](https://user-images.githubusercontent.com/592504/225321424-03f44e05-c0e9-450f-b152-c6664ebbe17a.png)

That would stop the line from wrapping at the default dialog width. @kswang1029 what do you think?

Reply:
We plan to have a new region/annotation dialog which would have two tabs, one for region props and the other for styling. So I guess we can leave it as it is for now. As for using a dropdown, that’s a good idea.

Reply:
I updated the selection to drop down. Please take a look at the updated code and ui.

Reply:
ui side looks good